packages/core/content-type-builder/admin/src/components/FormModal/attributes/attributeOptions.js

```diff
@@ -77,6 +77,43 @@ const attributeOptions = {
       defaultMessage: "You won't be able to create an entry if this field is empty",
     },
   },
+  step: {
+    name: 'step',
+    type: 'select',
+    intlLabel: {
+      id: getTrad('form.attribute.item.timeStep'),
+      defaultMessage: 'Time picker step (minutes)',
+    },
+    description: {
+      id: getTrad('form.attribute.item.timeStep.description'),
+      defaultMessage:
+        'The time interval (in minutes) between the items displayed in the time picker',
+    },
+    options: [
+      {
+        key: '__null_reset_value__',
+        value: '',
+        metadatas: {
+          intlLabel: {
+            id: 'components.InputSelect.option.placeholder',
+            defaultMessage: 'Choose here',
+          },
+        },
+      },
```

Comment:
What do you think about removing the `null` case and use `1` as default value? I feel like it would be better to explicit here, because a user might not know what `null` refers to in this case.

Reply:
The empty string allows to set the value to null in the ctb so you'll need to make some updates in the data manager provider.

Reply:
I removed the `null` option from the dropdown, it gets the value `1` by default.